doc/source/examples/extended_examples/index.rst

```diff
@@ -6,21 +6,23 @@ Extended examples
-+----------------------------+---------------------------------------------------------------------------------------------------------+
-| :ref:`krylov_example`      | Demonstrates the usage of the Krylov method for solving a customized harmonic analysis.                 |
-+----------------------------+---------------------------------------------------------------------------------------------------------+
-| :ref:`dash_example`        | Demonstrates the use of PyMAPDL with Dash.                                                              |
-+----------------------------+---------------------------------------------------------------------------------------------------------+
-| :ref:`executable_example`  | Create your own Python app with a command-line interface.                                               |
-+----------------------------+---------------------------------------------------------------------------------------------------------+
++----------------------------+--------------------------------------------------------------------------------------------+
+| **Name**                   | **Description**                                                                            |
++----------------------------+--------------------------------------------------------------------------------------------+
+| `Cart-Pole Simulation`_    | Demonstrates a reinforcement machine learning example using MAPDL through PyMAPDL.         |
++----------------------------+--------------------------------------------------------------------------------------------+
+| :ref:`extended_example01`  | Demonstrates the interoperability between PyMAPDL and the `gmsh <gmsh_>`_ meshing library. |
++----------------------------+--------------------------------------------------------------------------------------------+
+| :ref:`python_upf_examples` | Demonstrates the interoperability between user-programmable functions in MAPDL and Python. |
++----------------------------+--------------------------------------------------------------------------------------------+
+| :ref:`krylov_example`      | Demonstrates the usage of the Krylov method for solving a customized harmonic analysis.    |
++----------------------------+--------------------------------------------------------------------------------------------+
+| :ref:`dash_example`        | Demonstrates the use of PyMAPDL with Dash.                                                 |
++----------------------------+--------------------------------------------------------------------------------------------+
+| :ref:`executable_example`  | Create your own Python app with a command-line interface.                                  |
++----------------------------+--------------------------------------------------------------------------------------------+
+| :ref:`gui_example`         | Create your own GUI app with Python and PySide6                                            |
```

Comment:
```suggestion
| :ref:`executable_example`  | Demonstrates how to create your own Python app with a command-line interface.                                  |
+----------------------------+--------------------------------------------------------------------------------------------+
| :ref:`gui_example`         | Demonstrates how to create your own GUI app with Python and PySide6.                                            |
```
You will need to fix the columns--but the descriptions should all be constructed in a parallel fashion.